pkg/config/sample-config.txt

```diff
@@ -0,0 +1,26 @@
+version = 31
+credential.accessKey = xxx
+credential.secretKey = yyy    ## SecretKey cannot be changed
+region = us-east-1
+browser = on
+worm = on
+cache.drives = 
+cache.expiry = 111
+cache.maxuse = 100
```

Comment:
Max use is 80 default, we don't allow setting 100 AFAIK

Reply:
OK.
Just a note:
Almost all values in the sample configuration file is a random value and the "Check" function for each configuration parameter is an empty  stub at this point. 
If desired, I can collect the information on configuration parameter restrictions and requirements to implement the validation logic in the "Check" stubs.